enterprise/server/backends/pebble_cache/pebble_cache.go

```diff
@@ -415,6 +437,10 @@ func SetOptionDefaults(opts *Options) {
 	if opts.DeleteBufferSize == nil {
 		opts.DeleteBufferSize = &DefaultDeleteBufferSize
 	}
+	if opts.MinGCSFileSizeBytes == nil {
+		var maxInt int64 = math.MaxInt64
+		opts.MinGCSFileSizeBytes = &maxInt
```

Comment:
i don't know, given flag defaults and paranoia, on some level i'd be happy if this did the same for `0` in addition to `nil`